bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -1130,22 +1107,22 @@ <h2 class="section-title">{{ _('Mozilla Reps Council') }}</h2>
 
       <li class="vcard" itemscope itemtype="http://schema.org/Person">
         <figure class="headshot">
-          <a class="url" itemprop="url" rel="external" href="https://reps.mozilla.org/u/dricupello/">
-            {{ high_res_img('mozorg/about/leadership/adriano-cupello.jpg', {'alt': '', 'width': '160', 'height': '160', 'class': 'photo', 'itemprop': 'image'}) }}
```

Comment:
He's still on the council, the diff just shows this as a deletion because the lines shifted around. Adriano got moved down the file a bit (they're alphabetical by surname).